docs/articles/opinion/thoughts_on_governance_network_effects.md

```diff
@@ -6,84 +6,96 @@ _This article was originally posted in the [Aragon Blog](https://blog.aragon.one
+*   **Hardware**: [Ethereum](https://ethereum.org/) lets you pay for computation and storage on an as-needed basis by paying gas. Projects like [Golem](https://golem.network/), [Storj](https://storj.io/), [Truebit](https://truebit.io/), and others aim to further commoditize hardware through decentralized networks. Products that build on these technologies cannot rely on specialized hardware as a key differentiator.
+*   **Software**: Open source software is commoditized, there is no difference between one instance of public code versus another instance of the same code. Since the only applications that make sense to run on a blockchain are those that benefit from trust-minimization, public source code is the standard.
+*   **Data**: Decentralized applications rely on publicly available data sources, otherwise there is no way for the public to validate transactions. Privacy technology such as [zk-Snarks](https://blog.ethereum.org/2016/12/05/zksnarks-in-a-nutshell/) may obscure public visibility, but the encrypted dataset that enables a shared state remains public.
 
 Products or services offered in a market like this will either compete on price, quickly driving profit margins down or seek out new ways to differentiate themselves.
 
 **One way to differentiate is to issue a token**, even if you don’t otherwise need one. Having a token allows you to create a network effect around your product, people who hold this token will have a vested interest and prefer your product over alternatives. They will be your biggest advocates, they will shill your product to all their friends, who will jump on the band wagon, and do the same to their friends… and on and on. Or maybe not…
 
-If the market is commoditized and your competitors have at best an equivalent product to yours, issuing a token and requiring it to be used in order to access your product works (sort of like an API key). However, by forcing the use of a token when your application doesn’t require one you create a worse user experience because the user’s effective cost is unpredictable. Now if a competitor offers the same product without the restriction, it isn’t equivalent, it is objectively better. Therefore, differentiation in this way may work to some degree, but is not particularly defensible.
+If the market is commoditized and your competitors have at best an equivalent product to yours, issuing a token and requiring it to be used in order to access your product works (sort of like an [API key](https://en.wikipedia.org/wiki/Application_programming_interface_key)). However, by forcing the use of a token when your application doesn’t require one you create a worse user experience because the user’s effective cost is unpredictable. Now if a competitor offers the same product without the restriction, it isn’t equivalent, it is objectively better. Therefore, differentiation in this way may work to some degree, but is not particularly **defensible**.
+
+#### Defensible tokenization
```

Comment:
These header changes are breaking changes because of how the css is structured and should not be changed, request was for editing links **only**